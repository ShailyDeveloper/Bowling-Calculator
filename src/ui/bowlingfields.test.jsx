import React from 'react';
import { configure, shallow } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import BowlHandler from './bowlingfields';
import '@testing-library/jest-dom';

configure({ adapter: new Adapter() });

describe('BowlDropDown', () => {
    it('should match its snapshot', () => {
        const wrapper = shallow(<BowlHandler></BowlHandler>);
        expect(wrapper.find(BowlHandler)).toMatchSnapshot();
    });
});