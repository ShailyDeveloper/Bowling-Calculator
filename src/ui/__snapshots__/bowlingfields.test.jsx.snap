// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BowlDropDown should match its snapshot 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): ShallowWrapper {
    Symbol(enzyme.__root__): [Circular],
    Symbol(enzyme.__unrendered__): <Bowlval />,
    Symbol(enzyme.__renderer__): Object {
      "batchedUpdates": [Function],
      "checkPropTypes": [Function],
      "getNode": [Function],
      "render": [Function],
      "simulateError": [Function],
      "simulateEvent": [Function],
      "unmount": [Function],
    },
    Symbol(enzyme.__node__): Object {
      "instance": null,
      "key": undefined,
      "nodeType": "function",
      "props": Object {
        "children": Array [
          <div>
            <label
                        style={
                                    Object {
                                                "margin": "10px",
                                              }
                        }
            >
                        <b>
                                    Strike 
                        </b>
            </label>
</div>,
          <div>
            <select
                        onChange={undefined}
                        style={
                                    Object {
                                                "margin": "10px",
                                                "width": "10%",
                                              }
                        }
            />
</div>,
        ],
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": <label
              style={
                            Object {
                                          "margin": "10px",
                                        }
              }
>
              <b>
                            Strike 
              </b>
</label>,
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": <b>
                Strike 
</b>,
              "style": Object {
                "margin": "10px",
              },
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": Array [
                  "Strike ",
                  undefined,
                ],
              },
              "ref": null,
              "rendered": Array [
                "Strike ",
                undefined,
              ],
              "type": "b",
            },
            "type": "label",
          },
          "type": "div",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": <select
              onChange={undefined}
              style={
                            Object {
                                          "margin": "10px",
                                          "width": "10%",
                                        }
              }
/>,
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": undefined,
              "onChange": undefined,
              "style": Object {
                "margin": "10px",
                "width": "10%",
              },
            },
            "ref": null,
            "rendered": null,
            "type": "select",
          },
          "type": "div",
        },
      ],
      "type": Symbol(react.fragment),
    },
    Symbol(enzyme.__nodes__): Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "function",
        "props": Object {
          "children": Array [
            <div>
              <label
                            style={
                                          Object {
                                                        "margin": "10px",
                                                      }
                            }
              >
                            <b>
                                          Strike 
                            </b>
              </label>
</div>,
            <div>
              <select
                            onChange={undefined}
                            style={
                                          Object {
                                                        "margin": "10px",
                                                        "width": "10%",
                                                      }
                            }
              />
</div>,
          ],
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": <label
                style={
                                Object {
                                                "margin": "10px",
                                              }
                }
>
                <b>
                                Strike 
                </b>
</label>,
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": <b>
                  Strike 
</b>,
                "style": Object {
                  "margin": "10px",
                },
              },
              "ref": null,
              "rendered": Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": Array [
                    "Strike ",
                    undefined,
                  ],
                },
                "ref": null,
                "rendered": Array [
                  "Strike ",
                  undefined,
                ],
                "type": "b",
              },
              "type": "label",
            },
            "type": "div",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": <select
                onChange={undefined}
                style={
                                Object {
                                                "margin": "10px",
                                                "width": "10%",
                                              }
                }
/>,
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": undefined,
                "onChange": undefined,
                "style": Object {
                  "margin": "10px",
                  "width": "10%",
                },
              },
              "ref": null,
              "rendered": null,
              "type": "select",
            },
            "type": "div",
          },
        ],
        "type": Symbol(react.fragment),
      },
    ],
    Symbol(enzyme.__options__): Object {
      "adapter": ReactSixteenAdapter {
        "options": Object {
          "enableComponentDidUpdateOnSetState": true,
          "legacyContextMode": "parent",
          "lifecycles": Object {
            "componentDidUpdate": Object {
              "onSetState": true,
            },
            "getChildContext": Object {
              "calledByRenderer": false,
            },
            "getDerivedStateFromError": true,
            "getDerivedStateFromProps": Object {
              "hasShouldComponentUpdateBug": false,
            },
            "getSnapshotBeforeUpdate": true,
            "setState": Object {
              "skipsComponentDidUpdateOnNullish": true,
            },
          },
        },
      },
      Symbol(enzyme.__providerValues__): undefined,
    },
    Symbol(enzyme.__providerValues__): Map {},
  },
  Symbol(enzyme.__unrendered__): null,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "checkPropTypes": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): undefined,
  Symbol(enzyme.__nodes__): Array [],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "legacyContextMode": "parent",
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getChildContext": Object {
            "calledByRenderer": false,
          },
          "getDerivedStateFromError": true,
          "getDerivedStateFromProps": Object {
            "hasShouldComponentUpdateBug": false,
          },
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
    Symbol(enzyme.__providerValues__): undefined,
  },
  Symbol(enzyme.__rootNodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "function",
      "props": Object {
        "children": Array [
          <div>
            <label
                        style={
                                    Object {
                                                "margin": "10px",
                                              }
                        }
            >
                        <b>
                                    Strike 
                        </b>
            </label>
</div>,
          <div>
            <select
                        onChange={undefined}
                        style={
                                    Object {
                                                "margin": "10px",
                                                "width": "10%",
                                              }
                        }
            />
</div>,
        ],
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": <label
              style={
                            Object {
                                          "margin": "10px",
                                        }
              }
>
              <b>
                            Strike 
              </b>
</label>,
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": <b>
                Strike 
</b>,
              "style": Object {
                "margin": "10px",
              },
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": Array [
                  "Strike ",
                  undefined,
                ],
              },
              "ref": null,
              "rendered": Array [
                "Strike ",
                undefined,
              ],
              "type": "b",
            },
            "type": "label",
          },
          "type": "div",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": <select
              onChange={undefined}
              style={
                            Object {
                                          "margin": "10px",
                                          "width": "10%",
                                        }
              }
/>,
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": undefined,
              "onChange": undefined,
              "style": Object {
                "margin": "10px",
                "width": "10%",
              },
            },
            "ref": null,
            "rendered": null,
            "type": "select",
          },
          "type": "div",
        },
      ],
      "type": Symbol(react.fragment),
    },
  ],
  Symbol(enzyme.__providerValues__): null,
}
`;
